demo/app/assets/stylesheets/application.css

```diff
@@ -1,11 +1,28 @@
 /*
  *= require_tree .
  *= require_self
- *= require @primer/css/dist/primitives.css
  *= require @primer/css/dist/color-modes.css
  *= require @primer/css/dist/base.css
  *= require @primer/css/dist/buttons.css
  *= require @primer/css/dist/layout.css
  *= require @primer/css/dist/utilities.css
  *= require @primer/css/dist/markdown.css
+ *= require @primer/primitives/tokens-next-private/css/base/size/size.css
+ *= require @primer/primitives/tokens-next-private/css/base/typography/typography.css
+ *= require @primer/primitives/tokens-next-private/css/functional/size/border.css
+ *= require @primer/primitives/tokens-next-private/css/functional/size/breakpoints.css
+ *= require @primer/primitives/tokens-next-private/css/functional/size/size-coarse.css
+ *= require @primer/primitives/tokens-next-private/css/functional/size/size-fine.css
+ *= require @primer/primitives/tokens-next-private/css/functional/size/size.css
+ *= require @primer/primitives/tokens-next-private/css/functional/size/viewport.css
+ *= require @primer/primitives/tokens-next-private/css/functional/typography/typography.css
+ *= require @primer/primitives/tokens-next-private/css/functional/themes/dark-colorblind.css
+ *= require @primer/primitives/tokens-next-private/css/functional/themes/dark-dimmed.css
+ *= require @primer/primitives/tokens-next-private/css/functional/themes/dark-high-contrast.css
+ *= require @primer/primitives/tokens-next-private/css/functional/themes/dark-tritanopia.css
+ *= require @primer/primitives/tokens-next-private/css/functional/themes/dark.css
+ *= require @primer/primitives/tokens-next-private/css/functional/themes/light-colorblind.css
+ *= require @primer/primitives/tokens-next-private/css/functional/themes/light-high-contrast.css
+ *= require @primer/primitives/tokens-next-private/css/functional/themes/light-tritanopia.css
+ *= require @primer/primitives/tokens-next-private/css/functional/themes/light.css
```

Comment:
should we include these files in the same location as the old import?

Reply:
I'm not following what you mean 🤔 elaborate?

Reply:
The `@primer/css/dist/*` requires should come after the `@primer/primitives/*` requires